packages/connect-thegraph/src/connector.ts

```diff
@@ -168,10 +168,7 @@ class ConnectorTheGraph implements IOrganizationConnector {
         orgAddress: organization.address.toLowerCase(),
       },
       callback,
-      async (result) => {
-        const apps = await parseApps(result, organization)
-        return apps[0]
-      }
+      (result) => parseApp(result, organization)
```

Comment:
@0xGabi Wondering if you tested this? Because the query is `ORGANIZATION_APPS` with a limit of `1` since we added the filters (the single app query was only working by address).

Reply:
Apology, I didn't check on my local the change as it was small and seems straight forward but I got confused with how we were handling `APP_BY_ADDRESS` query.